actions/xpubs/update.go

```diff
@@ -50,3 +50,45 @@ func (a *Action) update(c *gin.Context) {
 	contract := mappings.MapToXpubContract(xPub)
 	c.JSON(http.StatusOK, contract)
 }
```

Comment:
As in Get, we should mark the xpub endpoint as deprecated and maybe add info in description what endpoint should be used instead.